### PR TITLE
R brance

### PR DESCRIPTION
**Pull request by rokars**
added some code to review changes

### CODE REVIEW

**Review by WtfWhyMe — APPROVED** [review]
LGTM

File: src/main/java/ie/gmit/main.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 package ie.gmit;
 
 public class main {
+
+    public void test1() {
+        System.out.println("GACHIGASM");
```
Comment (WtfWhyMe) [inline src/main/java/ie/gmit/main.java:6]: WEEB

File: src/main/java/ie/gmit/main.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+    }
     System.out.println("hello testing");
```
Comment (rokars) [inline src/main/java/ie/gmit/main.java:8]: so bad

Comment (rokars) [inline src/main/java/ie/gmit/main.java:8]: so bad

Comment (rokars) [reply src/main/java/ie/gmit/main.java:8]: @WtfWhyMe 
GachiApprove

**Pull request merged**